src/App.css

```diff
@@ -0,0 +1,77 @@
+/* body */
+body {
+    font-family: sans-serif;
+    background: #f4f4f4;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
+    margin: 0;
+  }
+  
+input {
+    width: 100%;
+    padding: 10px;
+    border-radius: 5px;
+    border: 1px solid #ccc;
+    margin-bottom: 10px;
+    font-size: 1rem;
+    font-weight: 400;
+    color: #333;
+  }
+  .rectangle-container {
+    height: 500px;
+    width: 450px;
+    background-color: #fafafa; 
+    border-radius: 10px; 
+    box-shadow: 0 0 10px rgba(205, 205, 205, 0.5);  
+  }
+
+
+.title{
+    text-align: center;
+    font-size: 1.5rem;
+    font-weight: 500;
+    color: #333;
+    margin: 0;
+}
+
+.subtitle{
+    text-align: center;
+    font-size: 1rem;
+    font-weight: 400;
+    color: #333;
+    margin: 0;
+}
+
+.square-container {
+    width: 350px; 
+    height: 250px;
+    background-color: #f4f4f4; 
+    border-radius: 10px;
+    border: 4px dashed rgb(125, 212, 252);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.Or {
```

Comment:
Avoid the use of capital letters in the class names. Read this for more info https://stackoverflow.com/a/2345300